frontend/star-tides/src/index.html

```diff
@@ -1,20 +1,23 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>StarTides</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
   <link rel="preconnect" href="https://fonts.gstatic.com">
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <meta name="google-signin-scope" content="profile email">
-  <meta name="google-signin-client_id" content="587210876817-3hadkdurufi94lup1eknasgnvkfr6nol.apps.googleusercontent.com">
+  <meta name="google-signin-client_id"
+    content="587210876817-3hadkdurufi94lup1eknasgnvkfr6nol.apps.googleusercontent.com">
```

Comment:
I'll just remove this stuff for now from the frontend. The backend signin code can stay as-is since it works great. For the frontend, we'll use https://github.com/abacritt/angularx-social-login which should integrate nicely with our Angular codebase (instead of signin being HTML+JS we add separately).